app/services/podcasts/get_media_url.rb

```diff
@@ -33,6 +33,7 @@ def result_struct
     end
 
     def url_reachable?(url)
+      url = Addressable::URI.parse(url).normalize
```

Comment:
shouldn't you call `to_s` here as well? This way you just pass the string to `HTTParty` and don't have to change the tests

Reply:
Good point, @rhymes , I changed that.